src/test/java/seedu/address/testutil/ModelStub.java

```diff
@@ -48,7 +50,8 @@ public void setAddressBookFilePath(Path addressBookFilePath) {
 
     @Override
     public void addPerson(Person person) {
-        throw new AssertionError("This method should not be called.");
+        throw new AssertionError(
```

Comment:
Actually i think no need separate into 2 lines but doesnt matter haha

Reply:
I'll move it back, I think my intellij did this haha.